app/views/templates/script/convert2rhel_analyze.erb

```diff
@@ -5,7 +5,19 @@ model: JobTemplate
 job_category: Convert 2 RHEL
 provider_type: script
 kind: job_template
+- name: ELS
+  required: true
+  input_type: user
+  description: Use an Extended Lifecycle Support (ELS) add-on subscription
+  advanced: false
+  value_type: plain
+  options: "yes\r\nno"
```

Comment:
(optional) Could we do true/false like we already do somewhere else[1]

[1] - https://github.com/theforeman/foreman_leapp/blob/master/app/views/foreman_leapp/job_templates/leapp_upgrade.erb#L21